src/pages/account/logins/[type]/[id]/update.tsx

```diff
@@ -51,7 +51,7 @@ const getServerSideProps: GetServerSideProps = async (context) => {
     context,
     [
       tracingMiddleware(getTracer()),
-      rateLimitingMiddleware(getTracer(), logger, rateLimitinConfig),
+      rateLimitingMiddleware(getTracer(), logger, rateLimitingConfig),
```

Comment:
😅

Reply:
Wtf, on my side it's `rateLimitingMiddleware` 🤔

Reply:
Haha :)

Reply:
Yes it is :)
I just put "😅" because of my typo, your change is good